GreyMatter/profile_populator.py

```diff
@@ -8,15 +8,19 @@ def empty(variable):
             return True
 
     print('Welcome to Melissa. Let us generate your profile!')
-    print(' Press Enter for using default values.')
+    print('Press Enter for using default values.')
 
     name = raw_input('Your name: ')
     if empty(name):
         name = 'Tanay'
 
-    stt = raw_input('STT Engine (google/sphinx): ')
-    if empty(stt):
-        stt = 'google'
+    while(True):
+        stt = raw_input('STT Engine (google/sphinx): ')
+        if stt in ('google', 'sphinx', ''):
+            if empty(stt):
+                stt = 'google'
+            break
+        print('Invalid input. Please try again.')
```

Comment:
Can we please change this to `print('Invalid input, please enter either google or sphinx.')`?